### PR TITLE
ZOOmailing FAQ and Troubleshooting

### PR DESCRIPTION
**Pull request by dguzenko**

### CODE REVIEW

File: 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: ZOOlingual Integration
+taxonomy:
+    category: docs
+---
+
+On multilanguage sites, which have both ZOOmailing and ZOOlingual, newsletters preview and sending are available in the language the admin user specifies when he/she logs in the Joomla administrator.
```
Comment (miljan-aleksic) [inline 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md]: This part is not clear "the admin user specifies when he/she logs in the Joomla administrator"

Comment (dguzenko) [inline 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md]: fixed

File: 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: ZOOlingual Integration
+taxonomy:
+    category: docs
+---
+
+On multilanguage sites, which have both ZOOmailing and ZOOlingual, preview and sent newsletters are displayed in the Joomla admin language.
```
Comment (miljan-aleksic) [inline 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md]: Sorry, I still don't understand what we are referencing here.

Comment (ameli90) [inline 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md]: When we preview the newsletter that uses ZOOmailing tag, the newsletter will be displayed in the current language admin logg in

Comment (miljan-aleksic) [inline 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md]: Update it as: On multilanguage sites, which have both ZOOmailing and ZOOlingual, preview and sent newsletters are displayed in the current backend language.

Comment (dguzenko) [inline 02.extensions/zoomailing/02.troubleshooting/02.zoolingual-integration/docs.md]: Done

**Pull request merged**